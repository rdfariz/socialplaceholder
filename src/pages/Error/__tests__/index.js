import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Component from '../';

describe('src/pages/Error', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(
      <Component />
    );
    expect(tree).toMatchSnapshot();
  });
});
