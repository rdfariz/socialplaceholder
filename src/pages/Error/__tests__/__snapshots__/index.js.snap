// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/Error render 1`] = `
<Component>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <meta
      charSet="utf-8"
    />
    <title>
      Social Dashboard - Something went wrong. Please try again
    </title>
  </HelmetWrapper>
  <div>
    <div
      className="w-full p-2 mb-1"
    >
      <Component
        className="bg-gradient-to-r from-pink-500 to-blue-500 text-white"
        label=""
        subtitle="Something went wrong. Please try again"
        title="Oops!"
      />
    </div>
  </div>
</Component>
`;
